src/app/portlets/dot-edit-content/services/html/iframe-edit-mode.js.ts

```diff
@@ -47,5 +65,8 @@ export const EDIT_PAGE_JS = `
             });
         }
     })
+
+    window.${MODEL_VAR_NAME}.next(getModel());
+    window.getModel = getModel;
```

Comment:
In the "window" we need to make sure that getModel() function have a better naming because it could collide with something in the page, the name is too common.